### ISSUE TITLE
edited photos don't contain the EXIF data from the ogiginal

### ISSUE THREAD

**Issue by sunhasgothishaton** [flathub/org.kde.digikam#20]
When I edit an image then save an image (save changes, or save as new version), it doesn't save the EXIF data from the original photo to the new photo.

This is different to the appimage version, the edited photo I save contains the EXIF data from the original.

Is there anything I can do to fix this?

**Comment by ElXreno** [comment]
I can't reproduce it. Could you try to reproduce this problem on a clean profile?

**Comment by sunhasgothishaton** [comment]
Thanks for your reply and help.

Is it only $HOME/.var/app/org.kde.digikam that I need to delete to wipe the profile, so I can create a clean profile?

How do I get logs files for digikam?

**Comment by ElXreno** [comment]
> Is it only $HOME/.var/app/org.kde.digikam that I need to delete to wipe the profile, so I can create a clean profile?

Yes. Just remove this folder or rename if you have any important information.

> I have digikam file in $HOME/.config and $HOME/.local/share do I need to delete them too?

To be sure that it does not interfere, it is better to delete or temporarily rename these folders as well.

> Any others?

No, it should be enough.

> How do I get logs files for digikam?

You can collect the logs as follows:
`flatpak run org.kde.digikam 2>&1 | tee digikam.log`
Go ahead and try to reproduce the problem.

I think this is enough for the start.

**Comment by sunhasgothishaton** [comment]
Found what was causing this, it was a setting in the configuration.

I ran with a clean profile and it worked ok, saving EXIF to new photos. So I went through the configuration (old and new) and saw that writing to sidecar files only had been selected, not reading from them. so it was writing limited EXIF to the sidecar files, just not reading from them.

I unselected "write to sidecar files" and new saved photos now have the EXIF of the originals.

Oops. Thanks for your help with this.
